pkgs/applications/science/electronics/kicad/default.nix

```diff
@@ -55,6 +86,16 @@ stdenv.mkDerivation rec {
   # this breaks other applications in kicad
   dontWrapGApps = true;
 
+  postInstall = ''
+    lndir -silent ${i18n}/share $out/share
+  '' + optionalString enablelibs ''
+    lndir -silent ${symbols}/share $out/share
```

Comment:
This is not easily overridable. Consider doing something like
```nix
{
  symbols = mkLib "symbols" "1rjh2pjcrc3bhcgyyskj5pssm7vffrjk0ymwr70fb7sjpmk96yjk" {};
  postInstall = ''
    # ...
    lndir -silent $symbols/share $out/share
    # ...
  '';
}
```

Then you will be able to override symbols using `overrideAttrs`.